src/SFA.DAS.FindEmploymentSchemes.Web/Infrastructure/Sitemap.cs

```diff
@@ -0,0 +1,41 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Hosting;
+using AspNetCore.SEOHelper.Sitemap;
+using SFA.DAS.FindEmploymentSchemes.Web.Content;
+
+
+namespace SFA.DAS.FindEmploymentSchemes.Web.Infrastructure
+{
+    public static class Sitemap
+    {
+        private static bool done = false;
+
+        public static void Generate(IWebHostEnvironment env, Uri baseUri)
+        {
+            if (!done)
+            {
+                List<SitemapNode> nodes = new List<SitemapNode>();
+
+                nodes.AddRange(SchemesContent.Schemes.Select(x => new SitemapNode() {
+                    LastModified = DateTime.UtcNow,
```

Comment:
The last modified time for the page will be incorrect, but I don't think we need to worry, as at least [Google seems to ignore it](https://webmasters.stackexchange.com/questions/108491/what-does-url-lastmod-in-a-sitemap-really-mean), as it's often wrong.

We can revisit once we integrate automatic updates from Contentful.

Reply:
Removed optional last modified field